pkgs/applications/window-managers/i3/wk-switch.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, python2Packages }:
+
+let
+  pkgName = "i3-wk-switch";
+in
+python2Packages.buildPythonApplication rec {
+  name = "${pkgName}-${version}";
```

Comment:
Instead of the hash, please instead  use the date of the commit.

Reply:
@grahamc Like this?

Reply:
Perfect.

Reply:
You can remove the name attribute, buildPythonApplication will concat pname and version attributes. As your set is recursive (rec) you can move pkgName into the set and rename it to `pname`